examples/composed-auth/auth/authorizers.go

```diff
@@ -95,13 +95,18 @@ func HasRole(token string, scopes []string) (*models.Principal, error) {
 			isInScopes := false
 			claimedRoles := []string{}
 			for _, scope := range scopes {
+				isInScopes = false
```

Comment:
What motivates your change here?
The initial intent of this demo was precisely to show how to merge claims from several scopes.
If you break on first match, this defeats the purpose...

Reply:
I wanted to use the HasRole function but the function succeeds when the scope are [foo, bar] and the user just has [foo] role. Then I propose changes for the next people which copy/paste the function.
For each scope iteration it starts for failing if we don't find the same role claimed in the role loop. If the role is found, we take the next scope and start by failing it until we find the same role claimed, etc
Hope my explanation is clear